src/core/thread/network_diagnostic_tlvs.hpp

```diff
@@ -95,6 +95,7 @@ class Tlv : public ot::Tlv
         kQueryId             = OT_NETWORK_DIAGNOSTIC_TLV_QUERY_ID,
         kMleCounters         = OT_NETWORK_DIAGNOSTIC_TLV_MLE_COUNTERS,
         kVendorAppUrl        = OT_NETWORK_DIAGNOSTIC_TLV_VENDOR_APP_URL,
+        kNonPrfChannelMask   = OT_NETWORK_DIAGNOSTIC_TLV_NON_PRF_CHANNEL_MASK,
```

Comment:
Since it's about the TLV name here (and not about the value "channel mask" contained in the TLV), suggesting to update the names to match Thread spec name of the TLV:

```suggestion
        kNonPrfChannelsMask   = OT_NETWORK_DIAGNOSTIC_TLV_NON_PRF_CHANNELS_MASK,
```

But if people don't like it, we can also leave as it was!